src/schema/v2/homeView/sections/InfiniteDiscovery.ts

```diff
@@ -0,0 +1,9 @@
+import { HomeViewSection } from "."
+import { HomeViewSectionTypeNames } from "../sectionTypes/names"
+
+export const InfiniteDiscovery: HomeViewSection = {
+  id: "home-view-section-infinite-discovery",
+  featureFlag: "diamond_home-view-infinite-discovery",
+  type: HomeViewSectionTypeNames.HomeViewSectionCard,
+  requiresAuthentication: true,
```

Comment:
This section will act as a link to open Infinite Discovery so it is very simple. I foresee adding analytics properties as the project progresses. Please tell me if there are other properties that I should consider.

Reply:
Yeah I think you will likely want to add a resolver that returns the card's data before merging this.

Right now the card section is wired up correctly — but it doesn't return any of the card-y data. This means that over in Eigen this section won't even render because of [this early return](https://github.com/artsy/eigen/blob/89c48d20c04126bcc43f97b242b84f3878b9125a/src/app/Scenes/HomeView/Sections/HomeViewSectionCard.tsx#L39-L41).

So a minimal PR here would likely also include a resolver that looks something [like this](https://github.com/artsy/metaphysics/blob/15bd7d2a4fa31c10d0b78ee918ed65e491076d63/src/schema/v2/homeView/sections/GalleriesNearYou.ts#L19-L29).

Let me know if you want to pair, happy to do so.

Reply:
Thanks for the advice. I didn't understand how Eigen consumes sections when I drafted this PR, and assumed that Eigen would be in charge of defining its contents. I've added the resolver now.